app/graphql/types/digital_object/asset_type.rb

```diff
@@ -0,0 +1,9 @@
+module Types
+  module DigitalObject
+    class AssetType < Types::BaseObject
+      implements Types::DigitalObjectInterface
```

Comment:
@cgalarza Is it possible for the various digital object graphql types to inherit from a common digital object type that implements `Types::DigitalObjectInterface`, so you don't need to repeat `implements Types::DigitalObjectInterface` in each class?

Reply:
Potentially?

I think it balances out because either way you have to specify which class to inherit/implement. Creating an intermediate type doesn't really make it any clearer, it just adds another level of indirection.

Reply:
I just suggested it because it's slightly DRYer since the class needs to inherit from something anyway.  If you'd prefer not to though, I'm okay with it as is.

Reply:
👍